druid/src/widget/button.rs

```diff
@@ -110,33 +111,40 @@ impl<T: Data> Button<T> {
 }
 
 impl<T: Data> Widget<T> for Button<T> {
+    #[instrument(name = "Button", level = "trace", skip(self, ctx, event, _data, _env))]
     fn event(&mut self, ctx: &mut EventCtx, event: &Event, _data: &mut T, _env: &Env) {
         match event {
             Event::MouseDown(_) => {
                 ctx.set_active(true);
                 ctx.request_paint();
+                use tracing::info;
```

Comment:
add this to the top of the file

Reply:
Nice catch. This was supposed to be a change to help me debugging, I'm removing it.